spec/formatting.md

```diff
@@ -235,6 +242,24 @@ rather than the _expression_ in the _selector_ or _pattern_.
 
 _Pattern selection_ is not supported for _fallback values_.
 
+### Expression Attribute Resolution
+
+The only _expression attribute_ that MUST be supported is `@locale`.
+If its value is set by a _literal_,
+it MUST be a comma-delimited sequence of IETF BCP 47 language tags.
+If its value is set by a _variable_,
+it MUST be either a single IETF BCP 47 language tag,
+or a sequence of IETF BCP 47 language tags.
+
+The resolved value of a valid `@locale` _expression attribute_
+is a sequence of IETF BCP 47 language tags.
+If a `@locale` has an invalid value,
+emit a Resolution error and ignore the _expression attribute_ in further processing.
+
+Implementations MAY ignore all other _expression attributes_ during formatting.
```

Comment:
What does "ignore" mean? Not process them?

Reply:
Yes.